### DISCUSSION TITLE
pake缓存目录是什么，需要清除

### DISCUSSION THREAD

**Discussion by Novices666** [tw93/Pake#720] [Q&A] [answered]
pake缓存目录是什么呢？
我ssl证书关掉了（不使用https了），但是重新打包http的地址后，会提示

你的连接不是专用连接
攻击者可能试图从 网址 窃取你的信息(例如，密码、消息或信用卡)。
NET::ERR_CERT_AUTHORITY_INVALID

在浏览器的话，直接清除缓存就好，但是咱们这个我一直没找到缓存目录或者ssl证书删除入口

**Comment by tw93** [comment]
教你一招，用一个卸载软件的工具来看，就知道在哪儿，基本在Library

**Reply by Novices666** [reply]
感谢！

**Comment by Novices666** [comment, marked as answer]
使用everything搜项目名，找到了
windows在
C:\Users\用户名\AppData\Roaming\项目名
